department-of-veterans-affairs/caseflow
# Assignments table

Connects #5821 

### Description
This PR displays the assignments data from the algorithm.

### Testing Plan
1. Create a schedulePeriod
1. Go to hearings/schedule/build/upload/{schedulePeriodId}
1. Confirm page displays correctly
